fe/fe-core/src/main/java/org/apache/doris/statistics/StatisticsAutoAnalyzer.java

```diff
@@ -70,6 +70,7 @@ protected void runAfterCatalogReady() {
         if (!StatisticsUtil.statsTblAvailable()) {
             return;
         }
+        analyzePeriodically();
```

Comment:
perroid analyze should in a serperate thread

Reply:
I quite agree with this, but this is a bug-fix pr other than refacor